github/VisualStudio
# Essentials Connect to GitHub

- Depends on #2360

### What this PR does

Add a `Connect to GitHub` button to the `Team Explorer > Connect> Manage Connections` menu. This command lets users add a GitHub / GitHub Enterprise connection or log in as a different user.

![image](https://user-images.githubusercontent.com/11719160/64872737-9efa5380-d63f-11e9-83d8-3ad51a6c857f.png)

### How to test

1. Install `GitHub Essentials` extension (make sure `GitHub for Visual Studio` isn't installed)
2. Open `Team Explorer` tool window
3. Click `Manage Connections` tool tray button
4. Click `Manage Connections > Connect to GitHub...`
5. Select `GitHub` or `GitHub Enterprise`
6. Enter credentials or `Sign in with your browser`

![image](https://user-images.githubusercontent.com/11719160/64873669-7c693a00-d641-11e9-9611-f82b39a84df8.png)
